src/Configurator/CopyFromRecipeConfigurator.php

```diff
@@ -66,6 +66,8 @@ private function copyFile(string $to, string $contents, bool $executable)
         if ($executable) {
             @chmod($to, fileperms($to) | 0111);
         }
+
+        $this->write('Copying file "'. basename($to) .'" to "'. $to.'"');
```

Comment:
I would reuse the work done in https://github.com/sensiolabs/SensioGeneratorBundle/pull/397 (probably to be adapted) for the way we display the information. See https://github.com/sensiolabs/SensioGeneratorBundle/pull/397/files#diff-0e445b5f16f472d999100a19e18c4e33R73 specifically.